packages/tldraw/src/lib/shapes/geo/GeoShapeUtil.tsx

```diff
@@ -167,6 +168,9 @@ export class GeoShapeUtil extends BaseBoxShapeUtil<TLGeoShape> {
 	}
 
 	override getFontFaces(shape: TLGeoShape): TLFontFace[] {
+		if (isEmptyRichText(shape.props.richText)) {
+			return EMPTY_ARRAY
+		}
```

Comment:
I added guards like this around text measurement (most geo shapes don't have text) but here they're only a partial solution. Parsing even "empty" text is slow because `getFontsFromRichText` is slow because it does a lot of work.